tests/solana_tests/call.rs

```diff
@@ -115,11 +115,11 @@ fn external_raw_call_with_returns() {
     let mut vm = build_solidity(
         r#"
         contract bar0 {
-            bytes4 private constant SELECTOR = bytes4(keccak256(bytes('test_bar(int64)')));
+            bytes8 private constant SELECTOR = bytes8(sha256(bytes('global:test_bar')));
 
             function test_other(bar1 x) public returns (int64) {
                 bytes select = abi.encodeWithSelector(SELECTOR, int64(7));
-                bytes signature = abi.encodeWithSignature("test_bar(int64)", int64(7));
+                bytes signature = abi.encodeWithSignature("global:test_bar", int64(7));
```

Comment:
This is not right. We can't suddenly require a `global:` prefix.

Reply:
I had not fully considered `abi.encodeWithSignature()`. We can prefix `global:` at runtime, but we can't do the camel-case conversion. I guess we have no choice but to leave this up to the solidity dev. In that case, we might as well let the solidity dev prefix `global:`.

Maybe you are right, we should document the discriminator including camel-case conversion.

Hopefully this will all go away once discriminators are in the idl.

Reply:
I've updated the documentation. Let me know what you think of it.

Reply:
Where are the changes?

Reply:
`docs/language/builtins.rst`. I described there how to use `abi.encodeWithSelector` and `abi.encodeWithSignature`.